README.md

```
@@ -31,86 +104,247 @@ yarn add lavacord/lavacord
+// Define your Lavalink nodes
+const nodes = [{ id: "1", host: "localhost", port: 2333, password: "youshallnotpass" }];
 
-// Initilize the Manager with all the data it needs
+// Create the manager with your bot's user ID and a send function
 const manager = new Manager(nodes, {
-    user: client.user.id, // Client id
-    send: (packet) => {
-        // this needs to send the provided packet to discord's WS using the method from your library.
-        // use the bindings for the discord library you use if you don't understand this
-    }
+	user: client.user.id, // Your bot's user ID
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)